{% extends 'base.html' %}
{% load static %}

{% block content %}

<div class="container mt-5">
  <h1 class="text-center mb-4">Database and Bioinformatic Tools Used</h1>
  
  <!-- NCBI References -->
  <div class="mb-4">
    <h2>NCBI Pathogen Detection Database</h2>
    <p>
      The NCBI Pathogen Detection Project. 2016. National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894, USA.
      <br>
      Available from: <a href="https://www.ncbi.nlm.nih.gov/pathogens/" target="_blank"> https://www.ncbi.nlm.nih.gov/pathogens/</a>
    </p>
  </div>

  <div class="mb-4">
    <h2>NCBI Sequence Read Archive (SRA)</h2>
    <p>
      The NCBI Sequence Read Archive. National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894, USA.
      <br>
      Available from: <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_blank"> https://www.ncbi.nlm.nih.gov/sra/</a>
    </p>
  </div>

  <!-- PubMLST References -->
  <div class="mb-4">
    <h2>PubMLST & Achtman 7-gene MLST Scheme</h2>
    <p>
      Jolley KA, Bray JE, Maiden MCJ. 2018. Open-access bacterial population genomics: BIGSdb software, the PubMLST.org website and their applications. Wellcome Open Res 24;3:124. 
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/30345391" target="_blank">30345391</a> | 
      DOI: <a href="https://doi.org/10.12688/wellcomeopenres.14826.1" target="_blank">10.12688/wellcomeopenres.14826.1</a>
    </p>
    <p>
      Achtman M, Wain J, Weill FX, Nair S, Zhou Z, Sangal V, et al. 2012. Multilocus sequence typing as a replacement for serotyping in Salmonella enterica. PLoS Pathog 8(6):e1002776. 
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/22737074" target="_blank">22737074</a> | 
      DOI: <a href="https://doi.org/10.1371/journal.ppat.1009040" target="_blank">10.1371/journal.ppat.1009040</a>
    </p>
  </div>

  <!-- Bacterial Antimicrobial Resistance References  -->
  <div class="mb-4">
    <h2>Bacterial Antimicrobial Resistance Reference Gene Database</h2>
    <p>
      Feldgarden M, Brover V, Gonzalez-Escalona N, Frye JG, Haendiges J, et al. 2021. AMRFinderPlus and the Reference Gene Catalog facilitate examination of the genomic links among antimicrobial resistance, stress response, and virulence. Sci Rep 11(1):12728.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/34135355" target="_blank">34135355</a> | 
      DOI: <a href="https://doi.org/10.1038/s41598-021-91456-0" target="_blank">10.1038/s41598-021-91456-0</a>
    </p>
    <p>
      Feldgarden M, Brover V, Fedorov B, et al. 2022. Curation of the AMRFinderPlus databases: applications, functionality and impact. Microb Genom 8(6):mgen000832.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/35675101" target="_blank">35675101</a> | 
      DOI: <a href="https://doi.org/10.1099/mgen.0.000832" target="_blank">10.1099/mgen.0.000832</a>
    </p>
  </div>

    <!-- VirulenceDB References -->
    <div class="mb-4">
        <h2>VirulenceDB</h2>
        <p>
            Han J, Tang H, Zhao S, Foley SL. 2024. Salmonella enterica virulence databases and bioinformatic analysis tools development. Sci Rep 14(1):25228.
          <br>
          PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/39448688/" target="_blank">39448688</a> | 
          DOI: <a href="https://doi.org/10.1038/s41598-024-74124-x" target="_blank">10.1038/s41598-024-74124-x</a>
        </p>
      </div>

  <!-- PlasmidFinder References -->
  <div class="mb-4">
    <h2>PlasmidFinder</h2>
    <p>
      Carattoli A, Zankari E, Garcia-Fernandez A, et al. 2014. In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence typing. Antimicrob Agents Chemother 58:3895−3903.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/24777092/" target="_blank">24777092</a> | 
      DOI: <a href="https://doi.org/10.1128/aac.02412-14" target="_blank">10.1128/AAC.02412-14</a>
    </p>
  </div>

  <!-- SPIFinder Reference -->
<div class="mb-4">
    <h2>SPIFinder</h2>
    <p>
      Roer L, Hendriksen RS, Leekitcharoenphon P, Lukjancenko O, Kaas RS, Hasman H, Aarestrup FM. 2016. 
      Is the evolution of <i>Salmonella enterica</i> subsp. <i>enterica</i> linked to restriction-modification systems? 
      <i>mSystems</i> 1(3):e00009-16.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/27822532" target="_blank">27822532</a> | 
      DOI: <a href="https://doi.org/10.1128/mSystems.00009-16" target="_blank">10.1128/mSystems.00009-16</a>
    </p>
  </div>

  <!-- INNUENDO wgMLST Scheme Reference -->
<div class="mb-4">
    <h2>INNUENDO wgMLST Scheme</h2>
    <p>
      Rossi M, Silva MSD, Ribeiro-Gonçalves BF, Silva DN, Machado MP, Oleastro M, Borges V, Isidro J, Viera L, Halkilahti J, Jaakkonen A, Palma F, Salmenlinna S, Hakkinen M, Garaizar J, Bikandi J, Hilbert F, Carriço JA. 2018. 
      INNUENDO whole genome and core genome MLST schemas and datasets for <i>Salmonella enterica</i>.
      <br>
      <a href="https://zenodo.org/records/1323684" target="_blank">Available from: https://zenodo.org/records/1323684</a>
    </p>
    <p>
      Alikhan NF, Zhou Z, Sergeant MJ, Achtman M. 2018. A genomic overview of the population structure of <i>Salmonella</i>. 
      <i>PLoS Genet</i> 14(4):e1007261.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/29621240" target="_blank">29621240</a> | 
      DOI: <a href="https://doi.org/10.1371/journal.pgen.1007261" target="_blank">10.1371/journal.pgen.1007261</a>
    </p>
  </div>

  <!-- SRA Toolkit Reference -->
<div class="mb-4">
    <h2>SRA Toolkit</h2>
    <p>
      SRA Toolkit Development Team. 2018. SRA Toolkit [Online].
      <br>
      <a href="https://github.com/ncbi/sra-tools" target="_blank">Available from: https://github.com/ncbi/sra-tools</a>
    </p>
  </div>
  
  <!-- pigz Reference -->
  <div class="mb-4">
    <h2>pigz</h2>
    <p>
      Adler M. pigz: a parallel implementation of gzip for modern multi-processor, multi-core machines [Online].
      <br>
      <a href="https://github.com/madler/pigz" target="_blank">Available from: https://github.com/madler/pigz</a>
    </p>
  </div>

  <!-- fastq-utils Reference -->
<div class="mb-4">
    <h2>fastq-utils</h2>
    <p>
      Fonseca N, Manning J. fastq-utils: validation and manipulation of FASTQ files, scRNA-seq barcode pre-processing and UMI quantification [Online].
      <br>
      <a href="https://github.com/nunofonseca/fastq_utils" target="_blank">Available from: https://github.com/nunofonseca/fastq_utils</a>
    </p>
  </div>
  
  <!-- Trimmomatic Reference -->
  <div class="mb-4">
    <h2>Trimmomatic</h2>
    <p>
      Bolger AM, Lohse M, Usadel B. 2014. Trimmomatic: a flexible trimmer for Illumina sequence data. <i>Bioinformatics</i> 30:2114−2120.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/24695404" target="_blank">24695404</a> | 
      DOI: <a href="https://doi.org/10.1093/bioinformatics/btu170" target="_blank">10.1093/bioinformatics/btu170</a>
    </p>
  </div>
  
  <!-- FastQC Reference -->
  <div class="mb-4">
    <h2>FastQC</h2>
    <p>
      Andrews S. 2010. FastQC: a quality control tool for high throughput sequence data [Online].
      <br>
      <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">Available from: http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>
    </p>
  </div>
  
  <!-- MultiQC Reference -->
  <div class="mb-4">
    <h2>MultiQC</h2>
    <p>
      Ewels P, Magnusson M, Lundin S, Käller M. 2016. MultiQC: summarize analysis results for multiple tools and samples in a single report. <i>Bioinformatics</i> 32(19):3047−3048.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/27312411" target="_blank">27312411</a> | 
      DOI: <a href="https://doi.org/10.1093/bioinformatics/btw354" target="_blank">10.1093/bioinformatics/btw354</a>
    </p>
  </div>
  
  <!-- Shovill Reference -->
  <div class="mb-4">
    <h2>Shovill</h2>
    <p>
      Seemann T. Shovill [Online].
      <br>
      <a href="https://github.com/tseemann/shovill" target="_blank">Available from: https://github.com/tseemann/shovill</a>
    </p>
  </div>
  
  <!-- SPAdes Reference -->
<div class="mb-4">
    <h2>SPAdes</h2>
    <p>
      Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. 
      SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. <i>J Comput Biol</i> 19:455−477.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/22506599" target="_blank">22506599</a> | 
      DOI: <a href="https://doi.org/10.1089/cmb.2012.0021" target="_blank">10.1089/cmb.2012.0021</a>
    </p>
  </div>
  
  <!-- QUAST Reference -->
  <div class="mb-4">
    <h2>QUAST</h2>
    <p>
      Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. QUAST: quality assessment tool for genome assemblies. <i>Bioinformatics</i> 29:1072−1075.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/23422339" target="_blank">23422339</a> | 
      DOI: <a href="https://doi.org/10.1093/bioinformatics/btt086" target="_blank">10.1093/bioinformatics/btt086</a>
    </p>
  </div>
  
  <!-- SeqSero2 Reference -->
  <div class="mb-4">
    <h2>SeqSero2</h2>
    <p>
      Zhang S, den Bakker HC, Li S, Chen J, Dinsmore BA, Lane C, Lauer AC, Fields PI, Deng X. 2019. 
      SeqSero2: rapid and improved <i>Salmonella</i> serotype determination using whole-genome sequencing data. <i>Appl Environ Microbiol</i> 85(23):e01746-19.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/31540993" target="_blank">31540993</a> | 
      DOI: <a href="https://doi.org/10.1128/AEM.01746-19" target="_blank">10.1128/AEM.01746-19</a>
    </p>
    <p>
      Zhang S, Yin Y, Jones MB, Zhang Z, Deatherage Kaiser BL, Dinsmore BA, Fitzgerald C, Fields PI, Deng X. 2015. 
      <i>Salmonella</i> serotype determination utilizing high-throughput genome sequencing data. <i>J Clin Microbiol</i> 53(5):1685−92.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/25762776" target="_blank">25762776</a> | 
      DOI: <a href="https://doi.org/10.1128/JCM.00323-15" target="_blank">10.1128/JCM.00323-15</a>
    </p>
  </div>
  
  <!-- SISTR Reference -->
  <div class="mb-4">
    <h2>SISTR</h2>
    <p>
      Yoshida CE, Kruczkiewicz P, Laing CR, Lingohr EJ, Gannon VP, Nash JH, Taboada EN. 2016. 
      The <i>Salmonella</i> In Silico Typing Resource (SISTR): an open web-accessible tool for rapidly typing and subtyping draft <i>Salmonella</i> genome assemblies. <i>PLoS One</i> 11:e0147101.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/26800248" target="_blank">26800248</a> | 
      DOI: <a href="https://doi.org/10.1371/journal.pone.0147101" target="_blank">10.1371/journal.pone.0147101</a>
    </p>
  </div>
  
  <!-- mlst Reference -->
  <div class="mb-4">
    <h2>mlst</h2>
    <p>
      Seemann T. mlst [Online].
      <br>
      <a href="https://github.com/tseemann/mlst" target="_blank">Available from: https://github.com/tseemann/mlst</a>
    </p>
  </div>
  
  <!-- chewBBACA Reference -->
  <div class="mb-4">
    <h2>chewBBACA</h2>
    <p>
      Silva M, Machado MP, Silva DN, Rossi M, Moran-Gilad J, Santos S, Ramirez M, Carriço JA. 2018. 
      chewBBACA: A complete suite for gene-by-gene schema creation and strain identification. <i>Microb Genom</i> 4(3):e000166.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/29543149" target="_blank">29543149</a> | 
      DOI: <a href="https://doi.org/10.1099/mgen.0.000166" target="_blank">10.1099/mgen.0.000166</a>
    </p>
  </div>
  
  <!-- cgmlst-dists Reference -->
  <div class="mb-4">
    <h2>cgmlst-dists</h2>
    <p>
      Seemann T. cgmlst-dists [Online].
      <br>
      <a href="https://github.com/tseemann/cgmlst-dists" target="_blank">Available from: https://github.com/tseemann/cgmlst-dists</a>
    </p>
  </div>
  
  <!-- ReporTree Reference -->
  <div class="mb-4">
    <h2>ReporTree</h2>
    <p>
      Mixão V, Pinto M, Sobral D, Di Pasquale A, Gomes JP, Borges V. 2023. 
      ReporTree: a surveillance-oriented tool to strengthen the linkage between pathogen genetic clusters and epidemiological data. <i>Genome Med</i> 15(1):43.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/37322495" target="_blank">37322495</a> | 
      DOI: <a href="https://doi.org/10.1186/s13073-023-01196-1" target="_blank">10.1186/s13073-023-01196-1</a>
    </p>
  </div>
  
  <!-- GrapeTree Reference -->
  <div class="mb-4">
    <h2>GrapeTree</h2>
    <p>
      Zhou Z, Alikhan NF, Sergeant MJ, Luhmann N, Vaz C, Francisco AP, Carrico JA, Achtman M. 2018. 
      GrapeTree: visualization of core genomic relationships among 100,000 bacterial pathogens. <i>Genome Res</i> 28:1395−1404.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/30049790" target="_blank">30049790</a> | 
      DOI: <a href="https://doi.org/10.1101/gr.232397.117" target="_blank">https://doi.org/10.1101/gr.232397.117</a>
    </p>
  </div>
  
  <!-- AMRFinderPlus Reference -->
  <div class="mb-4">
    <h2>AMRFinderPlus</h2>
    <p>
      Feldgarden M, Brover V, Gonzalez-Escalona N, Frye JG, Haendiges J, Haft DH, Hoffmann M, Pettengill JB, Prasad AB, Tillman GE, Tyson GH, Klimke W. 2021. 
      AMRFinderPlus and the Reference Gene Catalog facilitate examination of the genomic links among antimicrobial resistance, stress response, and virulence. <i>Sci Rep</i> 11(1):12728.
      <br>
      PubMed: <a href="https://pubmed.ncbi.nlm.nih.gov/34135355" target="_blank">34135355</a> | 
      DOI: <a href="https://doi.org/10.1038/s41598-021-91456-0" target="_blank">10.1038/s41598-021-91456-0</a>
    </p>
  </div>
  
  <!-- ABRicate Reference -->
  <div class="mb-4">
    <h2>ABRicate</h2>
    <p>
      Seemann T. ABRicate [Online].
      <br>
      <a href="https://github.com/tseemann/abricate" target="_blank">Available from: https://github.com/tseemann/abricate</a>
    </p>
  </div>
  
  <!-- R & R Packages Reference -->
<div class="mb-4">
    <h2>R & R Packages</h2>
    <p>
      R Core Team. 2016. R: a language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.
      <br>
      <a href="https://www.R-project.org/" target="_blank">Available from: https://www.R-project.org/</a>
    </p>
    <p>
      Barrett T, Dowle M, Srinivasan A, Gorecki J, Chirico M, Hocking T, Schwendinger B. 2024. data.table: Extension of 'data.frame'. [Online].
      <br>
      <a href="https://r-datatable.com" target="_blank">Available from: https://r-datatable.com</a>
    </p>
    <p>
      Wickham H, Averick M, Bryan J, Chang W, McGowan LD, François R, Grolemund G, Hayes A, Henry L, Hester J, Kuhn M, Pedersen TL, Miller E, Bache SM, Müller K, Ooms J, Robinson D, Seidel DP, Spinu V, Takahashi K, Vaughan D, Wilke C, Woo K, Yutani H. 2019. 
      Welcome to the tidyverse. <i>J Open Source Softw</i> 4(43): 1686.
      <br>
      DOI: <a href="https://doi.org/10.21105/joss.01686" target="_blank">10.21105/joss.01686</a>
    </p>
  </div>
  
</div>

{% endblock %}
